api.js

```diff
@@ -195,7 +195,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
```

Comment:
@sindresourhus: I kind of see where @jamestalmage is comming from, the some things recursive some not is a little confusing.

I'd like to purpose that we do something like: 

> **/test.js **/test-*.js test/**/*.js, **/*.spec.js **/*.test.js